conf/rails-pgsql

```diff
@@ -43,7 +43,10 @@ EOF
 
 chmod +x /usr/local/rbenv/rbenv.d/rehash/gem-symlinks.bash
 
-RBY_VER=2.7.6
+RBY_VER=3.1.4
```

Comment:
It might be best to only set `RBY_VER` if it isn't already set. That way if you set it in the individual `Makefile`, then it will be honored here.

I.e. something like this:
```
RBY_VER=${RBY_VER:-3.1.4}
```